src/functionaltest/java/com/ericsson/ei/frontend/pageobjects/SubscriptionPage.java

```diff
@@ -258,23 +251,27 @@ public boolean clickExpandButtonByXPath(String loc) {
     }
 
     public int countElements(String id) {
-        return driver.findElements(By.id(id)).size();
+        return driver.findElements(By.id(id))
+                     .size();
     }
 
     public void clickViewButtonByXPath(String XPath) {
-        // The row indicates weather or not the del / view and edit buttons has moved down to
+        // The row indicates weather or not the del / view and edit buttons has moved
+        // down to
```

Comment:
Another odd linebreak.

Reply:
Not my change. Had to rebase to latest code to solve the integrations test issues due to EI Backend changes. and got this into my PR.